clouddq/runners/dbt/dbt_utils.py

```diff
@@ -149,3 +149,13 @@ def get_bigquery_dq_summary_table_name(
         dbt_dataset = extract_dbt_env_var(dbt_dataset)
     dq_summary_table_name = f"{dbt_project}.{dbt_dataset}.{table_name}"
     return dq_summary_table_name
+
+
+def get_dbt_invocation_id(dbt_path: Path) -> str:
+
+    logger.debug("dbt path is %s", dbt_path)
+    # JSON file
+    with open(f"{dbt_path}/target/manifest.json") as f:
+        # Reading from file
+        data = json.loads(f.read())
+        return data["metadata"]["invocation_id"]
```

Comment:
we should log a useful error message if invocation_id is not found (Dbt hasn't run yet)

Reply:
Logged the error message, with the error information